requirements.txt

```diff
@@ -39,11 +43,12 @@ pyparsing==2.2.0
 python-dateutil==2.5.3
 python-env==1.0.0
 pytz==2017.2
-requests==2.13.0
+requests==2.16.5
 simplegeneric==0.8.1
 singledispatch==3.4.0.3
 six==1.10.0
 static3==0.7.0
 traitlets==4.2.1
 typing==3.6.1
+urllib3==1.21.1
 whitenoise==3.3.0
```

Comment:
Thank you. The reason why I ended upgrading them is because I was getting errors with the older versions of the requirements. I think the problem started when I rebased prior to working on the issue. Should I revert back to the previous requirements?